hack/e2e/run.sh

```diff
@@ -143,10 +145,62 @@ fi
 
 set -x
 pushd ./tests/e2e
-ginkgo . -v -p --nodes="${GINKGO_NODES}" --focus="${GINKGO_FOCUS}" --skip="${GINKGO_SKIP}" --report-dir="${ARTIFACTS}"
+ginkgo --focus="${GINKGO_FOCUS}" --skip="${GINKGO_SKIP}" --label-filter="${GINKGO_LABEL_FILTER}" . -v -p --nodes="${GINKGO_NODES}"  --report-dir="${ARTIFACTS}"
```

Comment:
why we added a filter for loadbalancer here

Reply:
Because I only want to run the `loadbalancer` test here and not the `ipv6 prefix` test. In general we could run both tests here and they will succeed but we will get nothing out of it.

Reply:
seems we now have a node test suit: https://github.com/kubernetes/cloud-provider-aws/blob/master/tests/e2e/nodes.go

this will be a regression that we skipped the "nodes" test suit when run e2e.
i think we shouldn't have this label-filter, and should run all tests we have.